tools/yaml2json.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import ruamel.yaml
+import json
+import glob
+import sys
+import os
+
+# grab the yaml files
+files = glob.glob('schemas/input/csl-*.yaml')
+
+for file in files:
+    ofn = os.path.join('build', os.path.splitext(file)[0] + '.json')
```

Comment:
this is going to need to be placed in the right directory for the gh-pages action to correctly work

Reply:
the python [pathlib](https://docs.python.org/3/library/pathlib.html) builtin library provides slightly more elegant ways to do these path operation. No need to switch, but just wanted to let you know if you're interested.